README.md

```diff
@@ -0,0 +1 @@
+[![Build Status](https://travis-ci.org/tobitron/tour-guide.svg?branch=master)](https://travis-ci.org/tobitron/tour-guide)
```

Comment:
I think this link is broken because it is case sensitive. Your usename and repo name should be capitalized.

Also, there should be a description of your app and a link to the live site here, at minimum.